webapp/public/locales/fr/strings.json

```diff
@@ -21,6 +21,7 @@
   "Cancel": "Annuler",
   "Cannot add {0} package": "Impossible d’ajouter le progiciel {0}",
   "Cannot load package: {0}": "Impossible de charger le paquet : {0}",
+  "Change Language": "Changer de langue",
   "Changes the value of the variable by this amount": "Modifie la valeur de la variable par ce montant",
```

Comment:
Yes I did it manually. The text actually doesn't render when the website is in French. I asked to Richard but he didn't know why. I would appreciate some explanations please.